azkaban-web-server/src/main/java/azkaban/webapp/servlet/LoginAbstractAzkabanServlet.java

```diff
@@ -409,6 +412,14 @@ protected void handleAjaxLoginAction(final HttpServletRequest req,
       cookie.setPath("/");
       resp.addCookie(cookie);
       getApplication().getSessionCache().addSession(session);
+      final Set<Session> sessionsOfSameIP =
+          getApplication().getSessionCache().findSessionsByIP(session.getIp());
+
+      // Check potential DDoS attack by bad hosts.
+      logger.info(
+          "Session id created for user '" + session.getUser().getUserId() + "' and ip " + session
+              .getIp() + ", " + sessionsOfSameIP.size() + " session(s) found from this IP "
+              + session.getIp());
```

Comment:
Do we need to repeat the ip information?

Reply:
good catch